scripts/roboclawtest.py

```diff
@@ -0,0 +1,19 @@
+# A short and sweet script to test communication with a single roboclaw motor controller.
+# usage 
+#   $ python roboclawtest.py 128
+
+from time import sleep
+import sys
+from os import path
+# need to add the roboclaw.py file in the path
+sys.path.append(path.join(path.expanduser('~'), 'osr_ws/src/osr-rover-code/ROS/osr/src'))
```

Comment:
Replace with relative path maybe?

Reply:
Meh. We already put this in the home directory, so it's internally consistent. I prefer the absolute path over the relative, because of that.